docs/config.json

```diff
@@ -8,6 +8,10 @@
           "title": "Get Started with Teleport",
           "slug": "/"
         },
+	{
+	  "title": "Documentation Overview",
+	  "slug": "/documentation-overview/"
```

Comment:
Should this be the default page that we show at goteleport.com/docs?

Reply:
The overview is based on what _used_ to be the default page. We changed the landing page to the previous "Install Teleport on a Linux Server" guide after a recommendation from Ev.